apps/addons/tests/test_models.py

```diff
@@ -2033,12 +2028,45 @@ def manifest(self, basename):
                             'addons', basename)
 
     def test_blacklisted_guid(self):
+        """New deletions won't be added to BlacklistedGuid but legacy support
+        should still be tested."""
         BlacklistedGuid.objects.create(guid='guid@xpi')
         with self.assertRaises(forms.ValidationError) as e:
             Addon.from_upload(self.get_upload('extension.xpi'),
                               [self.platform])
         eq_(e.exception.messages, ['Duplicate add-on ID found.'])
 
+    def test_existing_guid(self):
+        # Create deleted addon with same guid
```

Comment:
Would you rather have a docstring here instead of the comment? Also, we write comments like sentences: we end them with a final dot (just as docstrings).